### ISSUE TITLE
CPT support is not working with api v2

### ISSUE THREAD

**Issue by alexvandervegt** [WP-API/node-wpapi#114]
The following does not work with the v2 api:
```
wp.posts().type( type_name ): get posts of custom type type_name
```

When I read this article ( http://scottbolinger.com/custom-post-types-wp-api-v2/ ) it's clear why, the API endpoint changed.

**Comment by kadamwhite** [comment]
Correct, this is a known issue. Thanks for reporting, in hoping to have a new release out by new year's with much more support for v2!

**Comment by alexvandervegt** [comment]
@jasonphillips got this work on his APIv2 branch:

https://github.com/jasonphillips/wordpress-rest-api/tree/APIv2

**Comment by elyobo** [comment]
Looks like this is possibly working now, e.g. via `wp.types()`, but the 0.6.0 (current npm release) does not work (wrong endpoint in [types.js](https://github.com/kadamwhite/wordpress-rest-api/blob/v0.6.0-0/lib/types.js#L68), fixed in 645ba54).

Any chance of pushing up your more recent tags to npm @kadamwhite?

**Comment by elyobo** [comment]
#158 is a PR for the changes by @jasonphillips

**Comment by kadamwhite** [comment]
This should be resolved as of v0.8, which fully supports custom post types through automatic discovery, through bootstrapping or through `registerRoute`
